<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GdalToolsWidget</class>
 <widget class="QWidget" name="GdalToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Build Virtual Raster</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Input files</string>
       </property>
       <property name="buddy">
        <cstring>selectInputFilesButton</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="inputFilesEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectInputFilesButton">
         <property name="text">
          <string>Select...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Output file</string>
       </property>
       <property name="buddy">
        <cstring>selectOutputFileButton</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="outputFileEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectOutputFileButton">
         <property name="text">
          <string>Select...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="resolutionCheck">
       <property name="text">
        <string>&amp;Resolution</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="resolutionComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Highest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Average</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lowest</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="srcNoDataCheck">
       <property name="text">
        <string>&amp;Source No Data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="srcNoDataSpin">
       <property name="maximum">
        <number>65000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
